lib/internal/util/inspect.js

```diff
@@ -2175,6 +2178,485 @@ function formatBigIntNoColor(bigint, options) {
+  ['mediumvioletred', '#c71585'],
+  ['midnightblue', '#191970'],
+  ['mintcream', '#f5fffa'],
+  ['mistyrose', '#ffe4e1'],
+  ['moccasin', '#ffe4b5'],
+  ['navajowhite', '#ffdead'],
+  ['oldlace', '#fdf5e6'],
+  ['olivedrab', '#6b8e23'],
+  ['orangered', '#ff4500'],
+  ['orchid', '#da70d6'],
+  ['palegoldenrod', '#eee8aa'],
+  ['palegreen', '#98fb98'],
+  ['paleturquoise', '#afeeee'],
+  ['palevioletred', '#db7093'],
+  ['papayawhip', '#ffefd5'],
+  ['peachpuff', '#ffdab9'],
+  ['peru', '#cd853f'],
+  ['pink', '#ffc0cb'],
+  ['plum', '#dda0dd'],
+  ['powderblue', '#b0e0e6'],
+  ['rosybrown', '#bc8f8f'],
+  ['royalblue', '#4169e1'],
+  ['saddlebrown', '#8b4513'],
+  ['salmon', '#fa8072'],
+  ['sandybrown', '#f4a460'],
+  ['seagreen', '#2e8b57'],
+  ['seashell', '#fff5ee'],
+  ['sienna', '#a0522d'],
+  ['skyblue', '#87ceeb'],
+  ['slateblue', '#6a5acd'],
+  ['slategray', '#708090'],
+  ['slategrey', '#708090'],
+  ['snow', '#fffafa'],
+  ['springgreen', '#00ff7f'],
+  ['steelblue', '#4682b4'],
+  ['tan', '#d2b48c'],
+  ['thistle', '#d8bfd8'],
+  ['tomato', '#ff6347'],
+  ['turquoise', '#40e0d0'],
+  ['violet', '#ee82ee'],
+  ['wheat', '#f5deb3'],
+  ['whitesmoke', '#f5f5f5'],
+  ['yellowgreen', '#9acd32'],
+  ['rebeccapurple', '#663399'],
+]);
+
+const HASH_PATTERN = /^#([\dA-Fa-f]{2})([\dA-Fa-f]{2})([\dA-Fa-f]{2})([\dA-Fa-f]{2})?$/;
+const SMALL_HASH_PATTERN = /^#([\dA-Fa-f])([\dA-Fa-f])([\dA-Fa-f])([\dA-Fa-f])?$/;
+const RGB_PATTERN = /^rgba?\(\s*([+-]?\d*\.?\d+)\s*,\s*([+-]?\d*\.?\d+)\s*,\s*([+-]?\d*\.?\d+)\s*(,\s*([+-]?\d*\.?\d+)\s*)?\)$/;
+const HSL_PATTERN = /^hsla?\(\s*([+-]?\d*\.?\d+)\s*,\s*([+-]?\d*\.?\d+)%\s*,\s*([+-]?\d*\.?\d+)%\s*(,\s*([+-]?\d*\.?\d+)\s*)?\)$/;
+
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+// https://github.com/denoland/deno/blob/ece2a3de5b19588160634452638aa656218853c5/ext/console/01_console.js#L2739
+function parseCssColor(colorString) {
+  if (MapPrototypeHas(colorKeywords, colorString)) {
+    colorString = MapPrototypeGet(colorKeywords, colorString);
+  }
+  // deno-fmt-ignore
+  const hashMatch = RegExpPrototypeExec(HASH_PATTERN, colorString);
+  if (hashMatch != null) {
+    return [
+      Number(`0x${hashMatch[1]}`),
+      Number(`0x${hashMatch[2]}`),
+      Number(`0x${hashMatch[3]}`),
+    ];
+  }
+  // deno-fmt-ignore
+  const smallHashMatch = RegExpPrototypeExec(SMALL_HASH_PATTERN, colorString);
+  if (smallHashMatch != null) {
+    return [
+      Number(`0x${smallHashMatch[1]}${smallHashMatch[1]}`),
+      Number(`0x${smallHashMatch[2]}${smallHashMatch[2]}`),
+      Number(`0x${smallHashMatch[3]}${smallHashMatch[3]}`),
+    ];
+  }
+  // deno-fmt-ignore
+  const rgbMatch = RegExpPrototypeExec(RGB_PATTERN, colorString);
+  if (rgbMatch != null) {
+    return [
+      MathRound(MathMax(0, MathMin(255, Number(rgbMatch[1])))),
+      MathRound(MathMax(0, MathMin(255, Number(rgbMatch[2])))),
+      MathRound(MathMax(0, MathMin(255, Number(rgbMatch[3])))),
```

Comment:
Math.min ToNumbers its inputs.
```suggestion
      MathRound(MathMax(0, MathMin(255, rgbMatch[1]))),
      MathRound(MathMax(0, MathMin(255, rgbMatch[2]))),
      MathRound(MathMax(0, MathMin(255, rgbMatch[3]))),
```